TrigScint/src/TrigScint/TrigScintDigiProducer.cxx

```diff
@@ -92,14 +84,15 @@ void TrigScintDigiProducer::produce(framework::Event &event) {
       }
       if (particleMap[contrib.trackID].getPdgID() == 11 &&
           particleMap[contrib.trackID].getGenStatus() == 1) {
-        if (beamFrac.find(id) == beamFrac.end())
+        if (beamFrac.find(id) == beamFrac.end()) {
           beamFrac[id] = contrib.edep;
-        else
+        } else {
           beamFrac[id] += contrib.edep;
+        }
       }
     }
 
-    // for now, we take am energy weighted average of the hit in each stip to
+    // for now, we take am energy weighted average of the hit in each strip to
```

Comment:
then you might as well correct the typo am --> an 😄

Reply:
haha I missed that indeed